### PR TITLE
add cluster-task docs

### PR DESCRIPTION
**Pull request by AsfaMumtaz**

### CODE REVIEW

**Comment by stakater-user** [comment]
@AsfaMumtaz Image is available for testing. `docker pull docker-stakater-nexus.apps.prod.clusters.stakater.io/stakater/stakaterclouddocs:SNAPSHOT-PR-303-4971300a`

**Comment by stakater-user** [comment]
@AsfaMumtaz Image is available for testing. `docker pull docker-stakater-nexus.apps.prod.clusters.stakater.io/stakater/stakaterclouddocs:SNAPSHOT-PR-303-6e1b2019`

**Comment by stakater-user** [comment]
@AsfaMumtaz Image is available for testing. `docker pull docker-stakater-nexus.apps.prod.clusters.stakater.io/stakater/stakaterclouddocs:SNAPSHOT-PR-303-09f9b1b3`

File: content/sre/tekton/cluster-tasks.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+* MAJOR bump denotes incompatible API changes,
+* MINOR bump denotes backward compatible change or new feature
+* PATCH bump denotes bug fixes.
+
+Additional labels for pre-release and build metadata are available as extensions to the MAJOR.MINOR.PATCH format.
+
+## How
+The task utilizes GitVersion for creating version tags.
+
+## Version incrementing in GItversion:
```
Comment (rasheedamir) [inline content/sre/tekton/cluster-tasks.md]: @AsfaMumtaz there is capital I in GIT

File: content/sre/tekton/cluster-tasks.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+### Major Bump:
+To increment Major version **“+semver: major”**  or **“+semver: breaking”** needs to be added in one of the commit messages.*
```
Comment (rasheedamir) [inline content/sre/tekton/cluster-tasks.md]: what is purpose of * in the end of the line?

File: content/sre/tekton/cluster-tasks.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+## Semantic versioning:
+Given a version number in format 1.1.1 (Major.Minor.Patch)
+* MAJOR bump denotes incompatible API changes,
+* MINOR bump denotes backward compatible change or new feature
+* PATCH bump denotes bug fixes.
+
+Additional labels for pre-release and build metadata are available as extensions to the MAJOR.MINOR.PATCH format.
+
+## How
+The task utilizes GitVersion for creating version tags.
```
Comment (rasheedamir) [inline content/sre/tekton/cluster-tasks.md]: @AsfaMumtaz lets add link for the GitVersion

File: content/sre/tekton/cluster-tasks.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
+# stakater-create-git-tag-v1
+
+stakater-create-git-tag-v1 cluster task creates semantic versions for appplications.
+
+## Semantic versioning:
+Given a version number in format 1.1.1 (Major.Minor.Patch)
```
Comment (rasheedamir) [inline content/sre/tekton/cluster-tasks.md]: wouldn't it be better if we say x.y.z

**Comment by stakater-user** [comment]
@AsfaMumtaz Image is available for testing. `docker pull docker-stakater-nexus.apps.prod.clusters.stakater.io/stakater/stakaterclouddocs:SNAPSHOT-PR-303-0199deea`

**Review by rasheedamir — APPROVED** [review]

**Comment by stakater-user** [comment]
@rasheedamir Yikes! You better fix it before anyone else finds out! [Build](https://github.com/stakater/StakaterCloudDocs/commit//checks) has Failed!

**Comment by stakater-user** [comment]
@AsfaMumtaz Yikes! You better fix it before anyone else finds out! [Build](https://github.com/stakater/StakaterCloudDocs/commit//checks) has Failed!

**Pull request merged**